Vladyslav_Gurevych/HW6/Task3.py

```diff
@@ -0,0 +1,20 @@
+import re
+
+with open("text.txt", 'r') as file:
+    text = file.read()
+
+text = re.findall('[A-z]', text)
+formatted_text = ''.join(text).lower()
+alphabet = "abcdefghijklmnopqrstuvwxyz"
+result = {}
+for char in alphabet:
+    result[char] = 0
+
+for char in formatted_text:
+    for key, value in result.items():
+        if char == key:
+            value = value + 1
+            result[char] = value
+
+for key, value in result.items():
+    print(f'Letter "{key}" appears {value} times in this text.')
```

Comment:
Good. Alternative solution:

import re
with open("text.txt", 'r') as file:
    text = file.read()
regex = re.compile('[^A-z]')
result = list(regex.sub('', text).lower())
resulted_dict = {}.fromkeys(result)
for key in resulted_dict.keys():
    resulted_dict[key] = result.count(key)
print(resulted_dict)